x/pocketcore/types/session.go

```diff
@@ -110,14 +110,27 @@ func NewSessionNodes(sessionCtx, ctx sdk.Ctx, keeper PosKeeper, chain string, se
 	}
 	sessionNodes = make(SessionNodes, sessionNodesCount)
 	var node exported.ValidatorI
+	//unique address map
```

Comment:
NIT: update comment, "unique address map to avoid re-checking a pseudorandomly selected servicer"

Reply:
done